src/microbe_stage/PlayerHoverInfo.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using Godot;
+
+/// <summary>
+///   A system that manages reading what the player is hovering over.
+/// </summary>
+public class PlayerHoverInfo : Node
+{
+    private MicrobeCamera camera;
+    private CompoundCloudSystem cloudSystem;
+
+    /// <summary>
+    ///   All compounds the user is hovering over.
+    /// </summary>
+    public Dictionary<Compound, float> HoveredCompounds { get; private set; } = new Dictionary<Compound, float>();
```

Comment:
I think this could have the set removed as this should never be re-assigned, right?

Reply:
Indeed so.

Reply:
Wait nevermind, the compouds list specifically needs to be re-assigned every frame due to the `GetAllAvailableAt` call, so we need it be reassignable it seems like.

Reply:
Well at least the microbe list doesn't need to be reassigned, right?

Reply:
Yep, we only need it for the compounds list.